packages/buttons/src/styled/StyledButton.ts

```diff
@@ -124,18 +127,25 @@ const colorStyles = (
       }
     `;
   } else {
+    const borderColor =
+      props.isNeutral && !props.isDanger ? getColor('neutralHue', 300, props.theme) : baseColor;
+    const foregroundColor = props.isNeutral ? props.theme.colors.foreground : baseColor;
+    const hoverBorderColor = props.isNeutral ? baseColor : hoverColor;
+    const hoverForegroundColor = props.isNeutral ? foregroundColor : hoverColor;
+
     retVal = css`
-      border-color: ${!props.isBasic && baseColor};
+      border-color: ${!props.isBasic && borderColor};
       background-color: transparent;
-      color: ${baseColor};
+      color: ${foregroundColor};
 
       &:hover {
-        border-color: ${!props.isBasic && hoverColor};
-        background-color: ${rgba(baseColor as string, 0.08)};
-        color: ${hoverColor};
+        border-color: ${!props.isBasic && hoverBorderColor};
+        background-color: ${!props.isNeutral && rgba(baseColor as string, 0.08)};
+        color: ${hoverForegroundColor};
```

Comment:
I noticed that a button group (when toggled to use basic buttons) has a background color on hover and active:

<img src="https://user-images.githubusercontent.com/1811365/117053408-c0ed9180-accd-11eb-92db-8c497d5c0aee.gif" width="400" />

It looks like the same does not happen for when the buttons are neutral. Is this expected?

<img src="https://user-images.githubusercontent.com/1811365/117053413-c34feb80-accd-11eb-9cbb-728eacc5c650.gif" width='400' />

Reply:
Yes, this is a storybook snafu. The only way out of it is to clear the URL parameters and refresh the page. This does not happen with normal usage.

Reply:
Yes, this is a storybook snafu, which seems to be crumbling with certain combos. The solution is to clear URL parameters and refresh the page. This does not seem to be happening with normal usage.